fedimint-client/src/oplog.rs

```diff
@@ -24,6 +23,28 @@ use crate::db::{
     ChronologicalOperationLogKey, ChronologicalOperationLogKeyPrefix, OperationLogKey,
 };
 
+/// Json value using string representation as db encoding.
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct JsonStringed(pub serde_json::Value);
```

Comment:
Is this automatically transparent for serde? Otherwise we might be changing the serde/JSON encodig of `OperationLogEntry` by accident

Reply:
we can do #[serde(transparent)] but json transparent by default

Reply:
done